vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -73,6 +73,7 @@ LEDGER_EXPORT struct PaymentData {
 
 
 using PublisherBannerCallback = std::function<void(std::unique_ptr<ledger::PublisherBanner> banner)>;
+using WalletAddressesCallback = std::function<void(std::map<std::string, std::string> addresses)>;
```

Comment:
I'm seeing mixtures of `std::function<>` and `base::Callback`. Should we be using `base::Callback` if we're going to be adhering to Chromium dev procedures? (As well as base::Bind vice std::bind)

In fact, should we start implementing as much as possible out of base::?

Reply:
afaik we don't have base callback in native ledger, that's why we use bind